schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -0,0 +1,279 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "description": "The device state for an IP device",
+    "$ref": "#/definitions/DeviceShadowIP",
```

Comment:
It would improve maintainability if this schema was split up, so that each app that defines a shadow configuration is a separate file, and all namespaces (keys) are pointing to separate schemas. However, not a blocker.

Reply:
That would be useful. Something like:
```
schemas/
    └─ deviceShadow/
             └─ ipShadow/
                    ipShadow.json
                   └─ config/
                        └─ GPSPVT/
                           gps_pvt.json
                           gps_pvt-example.json
```